src/angularComponent/ngcOmniboxFieldDirective.js

```diff
@@ -30,7 +30,7 @@ export default function ngcOmniboxFieldDirective($document) {
           '{{::omnibox.autofocus === \'true\' || undefined}}');
       input.setAttribute('ng-disabled', 'omnibox.ngDisabled()');
 
-      tElement.html(input.outerHTML);
+      tElement[0].appendChild(input);
```

Comment:
Won't this cause two refresh cycles? One for clearing it out, then one for appending? As opposed to one?